introduction.md

```diff
@@ -1,4 +1,12 @@
 My name is Peter. It is my honor
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?